src/sage/databases/findstat.py

```diff
@@ -1,38 +1,28 @@
 # -*- coding: utf-8 -*-
 r"""
-FindStat - the Combinatorial Statistic Finder.
+FindStat - the search engine for combinatorial statistics and maps
 
-The FindStat database can be found at::
+The Sage interface to the FindStat database is::
```

Comment:
I prefer the old wording, or alternatively "The interface to".  Nota bene, throughout the file, "Sage" is still in lower case, so using "Sage" here introduces a very minor inconsistency.

Reply:
Thanks for the input. Changed to "Sage" throughout the file.